config.cmake

```diff
@@ -69,7 +65,6 @@ HHVM_EXTENSION(mongodb
  libmongoc/src/mongoc/mongoc-log.c
  libmongoc/src/mongoc/mongoc-matcher-op.c libmongoc/src/mongoc/mongoc-matcher.c
  libmongoc/src/mongoc/mongoc-memcmp.c
- libmongoc/src/mongoc/mongoc-opcode.c
```

Comment:
This is an actual source file. I'd suggest syncing your libbson and libmongoc lists with the shell snippet I use in https://github.com/mongodb/mongo-php-driver/blob/1553bbdf3dda62193389b7694af1be9c760c900b/config.m4#L233.